Portal/src/Datahub.Portal/gen-settings.ps1

```diff
@@ -0,0 +1,6 @@
+﻿$CurrentPath = Split-Path -Parent $MyInvocation.MyCommand.Path
+#get full path from $CurrentPath
+
+Import-Module $CurrentPath/../../../scripts/appsettings.psm1 -Force
+
+Export-AppSettings -SourceFile "template.appsettings.sand.json" -Target AppSettings -EnvironmentName test
```

Comment:
Should be `template.appsettings.json` no?

Reply:
I haven't fixed the individual scripts. Right now this assumes you run the gen-all-settings.ps1